src/main/java/ca/warp7/frc2023/subsystems/Elevator.java

```diff
@@ -0,0 +1,121 @@
+package ca.warp7.frc2023.subsystems;
+
+import static ca.warp7.frc2023.Constants.*;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxAlternateEncoder;
+import com.revrobotics.SparkMaxLimitSwitch;
+import com.revrobotics.SparkMaxLimitSwitch.Type;
+import com.revrobotics.SparkMaxPIDController;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Elevator extends SubsystemBase {
+    private static Elevator instance;
+
+    // returns the instance of the subsystem
+    public static Elevator getInstance() {
+        if (instance == null) instance = new Elevator();
+        return instance;
+    }
+
+    public static CANSparkMax createMasterSparkMAX(int deviceID) {
```

Comment:
Not worth having methods to create the motor controllers

Reply:
@SAC-4 if your are not able to do this then explain why

Reply:
I can't do this because we need the motors to be created inside a method because they can't be in the part of the code where we define classes I'm pretty sure.